docs/source/release_notes/lts/52_0.rst

```diff
@@ -0,0 +1,162 @@
+========
+52.0 LTS
+========
+
+The Avocado team is proud to present another release: Avocado version
+52.0, the second Avocado LTS version.
+
+What's new?
+===========
+
+When compared to the last LTS (v36), the main features introduced by
+this versions are:
+
+- TAP[2] version 12 support.
+- The download of assets in tests now allow for an expiration time.
+- Environment variables can be propagated into tests running on remote
+  systems.
+- The plugin interfaces have been moved into the
+  ``avocado.core.plugin_interfaces`` module.
```

Comment:
Or just nothing as plugins were already defined before and newcomers will get this info from standard documentation.